checks/evaluation/branch_protection_test.go

```diff
@@ -18,563 +18,1691 @@ import (
+			findings: []finding.Finding{
+				{
+					Probe:   "blocksDeleteOnBranches",
+					Outcome: finding.OutcomePositive,
+					Values: map[string]int{
+						"main": 1,
+					},
+				},
+				{
+					Probe:   "blocksForcePushOnBranches",
+					Outcome: finding.OutcomePositive,
+					Values: map[string]int{
+						"main": 1,
+					},
+				},
+				{
+					Probe:   "branchesAreProtected",
+					Outcome: finding.OutcomePositive,
+					Values: map[string]int{
+						"main": 1,
+					},
+				},
+				{
+					Probe:   "branchProtectionAppliesToAdmins",
+					Outcome: finding.OutcomePositive,
+					Values: map[string]int{
+						"main": 1,
+					},
+				},
+				{
+					Probe:   "dismissesStaleReviews",
+					Outcome: finding.OutcomePositive,
+					Values: map[string]int{
+						"main": 1,
+					},
+				},
+				{
+					Probe:   "requiresApproversForPullRequests",
+					Outcome: finding.OutcomePositive,
+					Values: map[string]int{
+						"main":                      1,
+						"numberOfRequiredReviewers": 1,
+					},
+				},
+				{
+					Probe:   "requiresCodeOwnersReview",
+					Outcome: finding.OutcomePositive,
+					Values: map[string]int{
+						"main": 1,
+					},
+				},
+				{
+					Probe:   "requiresLastPushApproval",
+					Outcome: finding.OutcomePositive,
+					Values: map[string]int{
+						"main": 1,
+					},
+				},
+				{
+					Probe:   "requiresUpToDateBranches",
+					Outcome: finding.OutcomePositive,
+					Values: map[string]int{
+						"main": 1,
+					},
+				},
+				{
+					Probe:   "runsStatusChecksBeforeMerging",
+					Outcome: finding.OutcomePositive,
+					Values: map[string]int{
+						"main": 1,
+					},
+				},
+				{
+					Probe:   "requiresPRsToChangeCode",
+					Outcome: finding.OutcomePositive,
+					Values: map[string]int{
+						"main": 1,
+					},
+				},
+			},
+			result: scut.TestReturn{
+				Score:        8,
+				NumberOfWarn: 1,
+				NumberOfInfo: 9,
+			},
 		},
 		{
 			name: "Branches are protected and require codeowner review, but file is not present",
-			expected: scut.TestReturn{
-				Error:         nil,
-				Score:         5,
-				NumberOfWarn:  3,
-				NumberOfInfo:  8,
-				NumberOfDebug: 0,
-			},
-			branch: &clients.BranchRef{
-				Name:      &branchVal,
-				Protected: &trueVal,
-				BranchProtectionRule: clients.BranchProtectionRule{
-					EnforceAdmins:           &trueVal,
-					RequireLastPushApproval: &falseVal,
-					RequireLinearHistory:    &trueVal,
-					AllowForcePushes:        &falseVal,
-					AllowDeletions:          &falseVal,
-					CheckRules: clients.StatusChecksRule{
-						RequiresStatusChecks: &falseVal,
-						UpToDateBeforeMerge:  &trueVal,
-						Contexts:             []string{"foo"},
-					},
-					RequiredPullRequestReviews: clients.PullRequestReviewRule{
-						Required:                     &trueVal,
-						DismissStaleReviews:          &trueVal,
-						RequireCodeOwnerReviews:      &trueVal,
-						RequiredApprovingReviewCount: &oneVal,
+			findings: []finding.Finding{
```

Comment:
this one doesn't really make sense anymore at this level. Given the codeowner change, we can only express positive/negative. Delete this one maybe?

Reply:
https://github.com/ossf/scorecard/pull/3759/commits/ef82d1225b946ef6a6f3f6f759043b399b371e10